index.js

```diff
@@ -20,6 +22,110 @@ app.get('/info', (req, res) => {
   res.send({ serverName: packageData.name, serverVersion: packageData.version });
 });
 
+// route for displaying all products
+app.get('/products/all', (req, res) => {
+
+  var jsonOutputObject = {
+    products : []
+  };
+
+  for(var i = 0; i < productData.products.length; i++) {
```

Comment:
You can use the [for..of loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) to improve code readability.

Reply:
Thanks @f4ww4z, that was a good sugestation :)